modules/@apostrophecms/module/index.js

```diff
@@ -439,13 +447,42 @@ module.exports = {
       // that point.
 
       async sendPage(req, template, data) {
+        console.log('sendPage - before', req.res.getHeader('Cache-Control'));
         await self.apos.page.emit('beforeSend', req);
+        console.log('sendPage - after', req.res.getHeader('Cache-Control'));
         await self.apos.area.loadDeferredWidgets(req);
         req.res.send(
           await self.apos.template.renderPageForModule(req, template, data, self)
         );
       },
 
+      setCacheControl(req, maxAge) {
+        if (typeof maxAge !== 'number') {
+          self.apos.util.warnDev('"maxAge" property must be defined as a number in the module cache option');
```

Comment:
should this check rather be done before? in the `apostrophe:modulesRegistered` event handler for instance?